django/contrib/postgres/search.py

```diff
@@ -92,14 +92,38 @@ def __init__(self, lhs, connector, rhs, config, output_field=None):
         super(CombinedSearchVector, self).__init__(lhs, connector, rhs, output_field)
 
 
-class SearchQuery(Value):
+class SearchQueryCombinable(object):
+    BITAND = '&&'
+    BITOR = '||'
+
+    def _combine(self, other, connector, reversed, node=None):
+        if not isinstance(other, SearchQueryCombinable) or not self.config == other.config:
```

Comment:
These lines are in `SearchVectorCombinable` as well. They're quite "defensive" coding - the easiest way to trigger it is to combine two queries with explicity set different languages. This would blow up at DB level with a different error.